CMakeLists.txt

```
@@ -57,6 +57,15 @@ if (SWIG_FOUND)
   else()
     message (STATUS "Searching for Ruby - found.")
   endif()
+
+  ########################################
+  # Include python
+  find_package(PythonLibs REQUIRED)
```

All language bindings should be optional, so we don't introduce a required dependency for users who don't care about them:

```suggestion
  find_package(PythonLibs QUIET)
```